back/config/schemas/settings.schema.json.erb

```diff
@@ -167,6 +167,12 @@
             "description": "What a tag unit should be called on this platform (e.g., department, theme, etc.). Input the singular form here with all lowercase letters. If left blank, this field defaults to 'tag'.",
             "$ref": "#/definitions/multiloc_string"
           },
+          "from_email": {
+            "title": "From email",
+            "description": "The email used in the from field when users receive emails from automated campaigns.",
```

Comment:
Probably a good idea to make it very explicit that this can only be configured in case the corresponding Second Line Support work as already happened and the customer has made the necessary DNS changes, or emails will not function at all.

Also, I think you're doing it for all email campaigns (or we should), not just the automated ones.

Reply:
Both good points. Thanks!